### PR TITLE
Issue #1042 Allow images to be used in the docs

### PR DESCRIPTION
**Pull request by hferentschik**
Addresses #1042

Adjustments to the docs build to be able to use images. Second commit is **not** to be merged. It is just added for testing purposes.

Please try locally as well as integrated in docs.openshift.org.

### CODE REVIEW

**Comment by hferentschik** [comment]
@Preeticp, this is probably also interesting for you.

**Comment by hferentschik** [comment]
@budhrg could you also please double check this. I think you have it all setup to build locally as well as against docs.openshift.org as well.

File: docs/config.rb [hunk 1]
```diff
@@ -9,6 +9,8 @@
 page '/*.json', layout: false
 page '/*.txt', layout: false
 
+set :relative_links, true
```
Comment (budhrg) [inline docs/config.rb]: :+1:  Nice, so this has enable rendering of `images/ci-servers.png` right?

Comment (hferentschik) [inline docs/config.rb]: It's part of it

**Comment by budhrg** [comment]
@hferentschik  Review with `make gen_docs && make serve_docs` and integerating with openshift-docs. The image is rendering as expected.

LGTM :+1:

**Comment by budhrg** [comment]
Merged and thanks!

**Pull request closed without merging**